ppl/zqd/ingest/archivepcap.go

```diff
@@ -171,9 +169,7 @@ func (p *archivePcapOp) runAnalyzer(ctx context.Context, group *errgroup.Group,
 	if err != nil {
 		return nil, nil, err
 	}
-	dr, err := newLogTailer(p.zctx, logdir, zio.ReaderOpts{
-		JSON: ndjsonio.ReaderOpts{TypeConfig: suricataTC, Warnings: p.warn},
-	})
+	dr, err := newLogTailer(p.zctx, logdir, zio.ReaderOpts{})
```

Comment:
Did you leave out the `Warnings: p.warn` option intentionally? (here & in the `legacyPcapOp` change below)?

Reply:
That option is ndjsonio-specific (and within ndjsonio, only used by the typed reader) which this PR is moving away from.